cypress/support/step_definitions/testSetup.js

```diff
@@ -111,6 +111,28 @@ Given('the environment has been set up for {string} tests', (test) => {
   cy.envConfigSetup();
 });
 
+/**
+ * @module TestSetupGlue
+ * @function Given the environment has been set up for '(.+)' tests(?: for '(.+)')
+ * @description Sets up the environment for the specified testType and scenarioType if specified
+ * @param {String} testType - The name of the test.
+ * @param {String} scenarioType - The name of the scenario which is optional.
+ * @example
+ * Given the environment has been setup for 'device'
+ * Given the environment has been setup for 'device' for 'version'
```

Comment:
as far as I know, the ask was not to create a new glue code . It is using the env has been setup for 'testtype' tests which is the existing glue code

Reply:
As per the discussion I had at offshore, the requirement was to create a new glue code as per the flowchart shared in the ticket, because the implementation is also somewhat different from the current testSetup.js
```
Given environment has been setup for 'Integrated Players’
Given environment has been setup for 'Integrated Players’? for '<Scenario Type>'
```

current glue code is something like -
`Given the environment has been set up for 'lifeCycleApi' tests`